src/main/java/org/example/Utils.java

```diff
@@ -78,4 +85,17 @@ public void transaction(Account account,Double amount,int receiversAccountNumber
         }
 
     }
+
+    public void showTransactionHistory(){
+        System.out.println("Your transaction history is:\n");
```

Comment:
Add a comment about what this function do

Reply:
corrected